docs/getting-started.md

```diff
@@ -250,12 +250,13 @@ kubectl apply -n dracon -f deploy/dracon/serviceaccount.yaml
 kubectl apply -n dracon -f deploy/dracon/role.yaml
 kubectl apply -n dracon -f deploy/dracon/rolebinding.yaml
 make cmd/draconctl/bin
+
+export DRACONCTL_MIGRATIONS_PATH='/etc/dracon/migrations/enrichment'
```

Comment:
is this value going to be passed to the container? I don't see any way in the code to propagate it. Also I don't think that it's needed when the dracon migrations container is used. maybe we can have it in the Dockerfile of the migrations container, since we are copying the migrations folder in there